static/app/views/issueDetails/traceTimeline/traceIssue.tsx

```diff
@@ -68,18 +71,44 @@ export function TraceIssueEvent({event}: TraceIssueEventProps) {
   );
 }
 
+// This function tries to imitate what getTitle() from utils.events does.
+// In that module, the data comes from the issues endpoint while in here
+// we grab the data from the events endpoint. A larger effort is
+// required in order to use that function directly since the data between
+// the two endpoint is slightly different.
+// For instance, the events endpoint could include a _metadata dict with
+// the title, subtitle and message.
+// We could also make another call to the issues endpoint  to fetch the metadata,
+// however, we currently don't support it and it is extremely slow
 function getTitleSubtitleMessage(event: TimelineEvent) {
   let title;
-  // XXX: This is not fully correct but it will make this first PR easier to review
-  const subtitle = event.transaction;
-  let message = event.message;
-  if (event['event.type'] === 'error') {
-    title = event.title.split(':')[0];
-  } else {
-    title = event.title;
-    // It is suspected that this value is calculated somewhere in Relay
-    // and we deconstruct it here to match what the Issue details page shows
-    message = event.message.replace(event.transaction, '').replace(title, '');
+  let subtitle = event.culprit || '';
```

Comment:
This moves us away from `event.transaction`:
<img width="542" alt="image" src="https://github.com/getsentry/sentry/assets/44410/ec24fe0a-a21f-414c-9530-dccff18e383c">

`getTitle` in `events.tsx` sets the value to these:
https://github.com/getsentry/sentry/blob/7a19d0bf37bec4ebdd3b2615e22f073e2e774003/static/app/utils/events.tsx#L136
https://github.com/getsentry/sentry/blob/7a19d0bf37bec4ebdd3b2615e22f073e2e774003/static/app/utils/events.tsx#L190